src/streamlit_app.py

```diff
@@ -69,7 +72,7 @@
 '''
 
 # Select Theme
-themes = Path('src/themes').iterdir()
+themes = Path('../src/themes').iterdir()
```

Comment:
⚪ Isn't it better to run the App on top of src folder and delete the `../` part of URL?